HW_3/18.py

```diff
@@ -0,0 +1,10 @@
+# Как и эта
+
+from random import randint
+n = int(input("Введите количество элементов: "))
+x = int(input("Введите число: "))
+lst = [randint(1, 7) for i in range(n)]
+for i in lst:
+    nearest = min(lst, key=lambda i:abs(i - x))
```

Comment:
Не совсем я понял, зачем вам цикл в этом случае.
```suggestion
nearest = min(lst, key=lambda i:abs(i - x))
```